keycmd/creds.py

```diff
@@ -22,12 +36,35 @@ def get_env(conf):
                 f" with user {src['username']}"
                 f" as it does not exist"
             )
-        if src.get("b64"):
-            password = b64(password)
-        env[key] = password
+        apply_b64 = src.get("b64", False)
+        format_string = src.get("format")
+        key_data[key] = (
+            src["credential"],
+            src["username"],
+            password,
+            apply_b64,
+            format_string,
+        )
+        expose(env, key, *key_data[key])
         vlog(
             f"exposing credential {src['credential']}"
             f" with user {src['username']}"
-            f" as environment variable {key} (b64: {src.get('b64', False)})"
+            f" as environment variable {key}"
+            f" (b64: {apply_b64}, format: {format_string})"
+        )
+
+    for alias, src in conf.get("aliases", {}).items():
+        key = key_data.get(src["key"])
+        if key is None:
+            error(f"MISSING alias key {src['key']}")
+        # re-use base data but replace apply_b64 and format_string
+        credential, username, password, _, _ = key
+        apply_b64 = src.get("b64", False)
```

Comment:
If you create an alias for a key that has b64 set to True, and don't set the b64 in the alias, wouldn't you expect the alias to also be b64?
So maybe use:
```python
credential, username, password, apply_b64, _ = key
apply_b64 = src.get("b64", apply_b64)
```